docs/bugs.json

```diff
@@ -1,4 +1,17 @@
 [
+    {
+        "uid": "SOL-2023-3",
+        "name": "BlockDeduplicatorVerbatimAssemblyItemBrokenComparison",
+        "summary": "Two blocks that are completly identical except for a difference in the data of a ``verbatim`` assembly item may wrongfully compare equal.",
+        "description": "The block deduplicator is an optimizer step which identifies equivalent assembly blocks and merges them into a single one. However when blocks contained ``verbatim`` data, their comparison was wrongfully executed, leading to the collapsing of assembly blocks which are identical but on the data of ``verbatim`` items. This could produce final bytecode which does not run as the intended solidity code. The bug can be avoided, in versions previous to the fix, by disabling the flag ``deduplicate`` in the optimizer details settings via the standard Json input interface.",
+        "link": "https://blog.soliditylang.org/2023/X/Y/Z/",
+        "introduced": "0.8.5",
+        "fixed": "0.8.23",
+        "severity": "low",
```

Comment:
Not entirely sure if ``very low`` wouldn't even be sufficient. Another case where it'd be much easier if we split this into likelihood and impact :-). (Likelihood is near zero, while impact is actually high)

Reply:
Call decision: we keep it at low.